### PR TITLE
Clean `darwinia-support` Part.1

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: frame/support/src/structs.rs [hunk 1]
```diff
@@ -20,62 +20,13 @@
...
-}
-
-#[derive(Clone, PartialEq, Eq, Encode, Decode, RuntimeDebug, MaxEncodedLen, TypeInfo)]
-pub enum LockFor<Balance, Moment> {
-	Common { amount: Balance },
-	Staking(StakingLock<Balance, Moment>),
-}
-
 #[derive(Clone, Default, PartialEq, Eq, Encode, Decode, RuntimeDebug, MaxEncodedLen, TypeInfo)]
 pub struct StakingLock<Balance, Moment> {
```
Comment (AsceticBear) [inline frame/support/src/structs.rs:31]: Why not move this to the staking pallet scope?

File: frame/wormhole/backing/ethereum/src/lib.rs [hunk 2]
```diff
@@ -578,10 +578,8 @@ pub mod pallet {
 				if is_ring { <Error<T>>::RingLockedNSBA } else { <Error<T>>::KtonLockedNSBA }
 			);
 			// // Checking redeemer have enough of balance to pay fee, make sure follow up transfer
-			// will success. ensure!(
-			// 	T::RingCurrency::usable_balance(redeemer) >= fee,
-			// 	<Error<T>>::FeeIns
-			// );
+			// will success.
```
Comment (AsceticBear) [inline frame/wormhole/backing/ethereum/src/lib.rs:581]: @xiaoch05 Any suggestion here?

File: frame/support/src/structs.rs (hunk 1 above)
Comment (AurevoirXavier) [reply frame/support/src/structs.rs:31]: Still in used.

File: frame/wormhole/backing/ethereum/src/lib.rs (hunk 2 above)
Comment (xiaoch05) [reply frame/wormhole/backing/ethereum/src/lib.rs:581]: Perhaps the fee's system here is not yet fully realized. cc @AurevoirXavier

Comment (AurevoirXavier) [reply frame/wormhole/backing/ethereum/src/lib.rs:581]: Yep.
Let's leave it here for now.

**Review by AsceticBear — APPROVED** [review]

**Pull request merged**